extensions/wikia/Discussions/EnableDiscussionsController.class.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Email\Fatal;
+
+class EnableDiscussionsController extends \WikiaController {
+
+	const ENABLE_DISCUSSIONS_NAVIGATION = 'wgEnableDiscussionNavigation';
+	const ENABLE_DISCUSSIONS = 'wgEnableDiscussion';
+
+	const P_SITE_ID = 'siteId';
+	const REASON = 'Enabling discussions!';
+
+	public function index() {
+		$this->request->isInternal();
```

Comment:
This call is only checking if the incoming request is internal. But nothing is being with the return value.